src/client/containers/ProfilePage/MentorProfile/MentorProfile.component.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+import { AppHeader } from '../../../components/Appheader/AppHeader.component';
+import { Avatar } from '../../../components/Avatar/Avatar';
+import mentor from '../../../assets/images/mentor.png';
+import Vector from '../../../components/Appheader/Vector.svg';
+import Label from '../../../components/Label/Tags/Label';
+import DropDown from '../../../components/Dropdown/DropDown';
+import Footer from '../../../components/footer/footer';
+import { Button } from '../../../components/Button/Button';
+import { TimeZoneDropDown } from '../../../components/TimeZone/TimeZone.component';
+import './MentorProfile.style.css';
+
+const timeZones = [
+  {
+    id: 1,
+    name: 'Europe/Copenhagen',
+  },
+  {
+    id: 2,
+    name: 'Europe/Dublin',
+  },
+  {
+    id: 3,
+    name: 'Europe/Budapest',
+  },
+  {
+    id: 4,
+    name: 'Australia/Sydney',
+  },
+];
```

Comment:
Now that I think about this, please put these zones _inside_ TimeZoneDropDown. Since they will always be the same.